transformer_engine/jax/__init__.py

```diff
@@ -2,5 +2,8 @@
 #
 # See LICENSE for license information.
 """Transformer Engine bindings for JAX"""
-from .fp8 import fp8_autocast
+from .fp8 import fp8_autocast, update_collections, update_fp8_metas
+from .module import DenseGeneral, LayerNormDenseGeneral, LayerNormMlpBlock, TransformerEngineBase
```

Comment:
Could we change the `LayerNormMlpBlock` to `LayerNormMLP` to be consistent with the pyTorch version?